xml/kernel_modules.xml

```diff
@@ -14,13 +14,72 @@
   </dm:docmanager>
  </info>
  <para>
-   
+   Although Linux is a monolithic kernel, it can be extended through kernel
+   modules. These are special objects that can be inserted into the kernel and
+   removed on demand. In practical terms, kernel modules make it possible to
+   add and remove drivers and interfaces that are not included in the kernel
+   itself. Linux provides several commands for managing kernel modules.
  </para>
  <sect1 xml:id="sec.mod.lsmod">
-  <title></title>
-
+  <title>Listing Loaded Modules with <command>lsmod</command> and <command>modinfo</command></title>
   <para>
-    
+    Use the <command>lsmod</command> command to view what kernel modules are
+    currently loaded. The output of the command may look as follows:
   </para>
+  <screen>$ lsmod
+          Module                  Size  Used by
+          snd_usb_audio         188416  2 
+          snd_usbmidi_lib        36864  1 snd_usb_audio
+          hid_plantronics        16384  0 
+          snd_rawmidi            36864  1 snd_usbmidi_lib
+          snd_seq_device         16384  1 snd_rawmidi
+          fuse                  106496  3 
+          nfsv3                  45056  1 
+          nfs_acl                16384  1 nfsv3</screen>
+  <para>
+    The output is divided into three columns. The "Module" column lists the names
```

Comment:
Maybe use literal or guimenu to mark up the column names..? (at the very least, the should be in `<quote/>` tags instead of bare quote characters.)